common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/CertBasedAuthFactory.java

```diff
@@ -90,7 +91,12 @@ protected CertBasedAuthFactory(@NonNull final Activity activity,
      * @param callback logic to run after an AbstractCertBasedAuthChallengeHandler is chosen.
      */
     public void createCertBasedAuthChallengeHandler(@NonNull final CertBasedAuthChallengeHandlerCallback callback) {
-        final ICertBasedAuthTelemetryHelper telemetryHelper = new CertBasedAuthTelemetryHelper();
+        final ICertBasedAuthTelemetryHelper telemetryHelper;
+        if (mActivity instanceof AuthorizationActivity) {
```

Comment:
What is the other possible type for activity here? and in what cases do we hit that route?

Reply:
This class gets the activity instance from AzureActiveDirectoryWebViewClient, which takes in any Activity as a parameter. This makes it possible for an activity other than AuthorizationActivity to be passed here, so in the (albeit unlikely) event that this does happen in the future, I'd rather handle this silently than assume and risk a crash.